numpy/ma/core.py

```diff
@@ -7818,8 +7818,10 @@ def dump(a, F):
 
     """
     if not hasattr(F, 'readline'):
-        F = open(F, 'w')
-    return pickle.dump(a, F)
+      with open (F, 'w') as F:
+        pickle.dump(a, F)
+    else:
+      pickle.dump(a, F)
```

Comment:
Style nit: 4-space indents, please.

There are other functions in this module that need the same fix, if you're going to fix this one.

Reply:
Ah, my bad.

I will fix any others I find. 

I see now that I should have 'wb' instead of just 'w', as per #5491. Is there a way for me to change this without creating another pull request?

Reply:
Leave the `wb` to a different PR - it's a separate problem, and I don't think we should fix it anyway.

Reply:
> Is there a way for me to change this without creating another pull request?

I assume you're editing in the github web UI? You can navigate to you branch, and hit the edit button on the files there. It'll create a new commit, but we can squash them all together when you're done.

Reply:
Sorry, I didn't see your replies to this comment before I changed the `wb`. Yes, I'm using the Github web UI - I think I got it to work, thanks!